src/env.zig

```diff
@@ -61,6 +61,25 @@ pub const OperatingSystem = enum {
     // wAsM is nOt aN oPeRaTiNg SyStEm
     wasm,
 
+    pub const names = @import("root").bun.ComptimeStringMap(
+        OperatingSystem,
+        &.{
+            .{ "windows", OperatingSystem.windows },
+            .{ "win32", OperatingSystem.windows },
+            .{ "win64", OperatingSystem.windows },
+            .{ "win_x64", OperatingSystem.windows },
+            .{ "darwin", OperatingSystem.mac },
+            .{ "macos", OperatingSystem.mac },
+            .{ "macOS", OperatingSystem.mac },
```

Comment:
this implies the entire list is case insensitive, meaning `Linux` will not work?

Reply:
added